pycaret/internal/tabular.py

```diff
@@ -5786,6 +5790,12 @@ def is_tree(e):
         raise TypeError(
             "feature parameter must be string containing column name of dataset."
         )
+        
+    # checking display_format parameter
+    plot_formats = [None, "streamlit"]
+    
+    if display_format not in plot_formats:
+        raise ValueError("display_format can only be None or \"streamlit\".")
```

Comment:
Please use single quotes `'` instead of escaping double quotes

Reply:
Done. Not sure why it's not passing all the checks though.